### PR TITLE
Functions

### PR DESCRIPTION
**Pull request by pjmiles**
started signup function. There is CROSS error issue

### CODE REVIEW

File: src/components/login/Login.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          <div className="input-container">
+            <label htmlFor="password"></label>
+            <input type="password" className="login-input" placeholder="password" id="password" required />
+          </div>
+          <div className="login-btn-container">
+            <button className="login-input-btn">
+              Login
+            </button>
+          </div>
+          <span className="login-note">Forgot password <Link to="/signup">click here</Link></span>
```
Comment (Asylcreek) [inline src/components/login/Login.js]: Go back to signup if I've forgotten my password?

File: src/components/signup/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      [e.target.name]: e.target.value.trim(),
+    }));
+  };
+
+  const handleSubmit = () => {
+    if (user.password !== user.password_confirmation) {
+      setErrMsg("Password does not match");
+      setSuccess(false);
+    } else {
+      postRequest();
```
Comment (Asylcreek) [inline src/components/signup/SignUp.js]: You using this before it is initialized?

File: src/components/signup/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    setUser((state) => ({
+      ...state,
+      [e.target.name]: e.target.value.trim(),
+    }));
+  };
+
+  const handleSubmit = () => {
+    if (user.password !== user.password_confirmation) {
+      setErrMsg("Password does not match");
+      setSuccess(false);
```
Comment (Asylcreek) [inline src/components/signup/SignUp.js]: Why is this here?

File: src/components/signup/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+    } else {
+      postRequest();
+      setSuccess(true);
```
Comment (Asylcreek) [inline src/components/signup/SignUp.js]: Why is this here?

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: src/components/login/Login.js (hunk 1 above)
Comment (pjmiles) [reply src/components/login/Login.js]: this will be adjusted I just used it to adjust the design

File: src/components/signup/SignUp.js (hunk 3 above)
Comment (pjmiles) [reply src/components/signup/SignUp.js]: oh not needed for sign met will be adjusted

File: src/components/signup/SignUp.js (hunk 2 above)
Comment (pjmiles) [reply src/components/signup/SignUp.js]: will rearrange this

File: src/components/signup/SignUp.js (hunk 4 above)
Comment (pjmiles) [reply src/components/signup/SignUp.js]: adjusted

File: src/components/signup/SignUp.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+      setErrMsg(err.message);
+      setSuccess(false);
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    if (user.password !== user.password_confirmation) {
+      setErrMsg("Password does not match");
+      setSuccess(false)
```
Comment (Asylcreek) [inline src/components/signup/SignUp.js]: Why?

Comment (pjmiles) [reply src/components/signup/SignUp.js]: I actually dont need this, let me allow backend handle it and I print the response

File: src/components/login/Login.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    <div className="login">
+      <div className="login-split login-left">
+        <div className="login-centered">
+          <h2>Twitter</h2>
+          <FaTwitter className="login-logo" />
+        </div>
+      </div>
+
+      <div className="login-split login-right">
+          <h2 className="login-header">Login</h2>
```
Comment (Asylcreek) [inline src/components/login/Login.js:16]: You can use `h3` instead

**Review by Asylcreek — APPROVED** [review]
